### PR TITLE
fix some module dependencies

### PR DESCRIPTION
**Pull request by nathantsoi**
@NathanWalker, this is more fyi than to pull in. The app renders, but tests are failing and no changes were made to the nativescript side of the app.

Nonetheless, I hope it's helpful.

### CODE REVIEW

**Comment by NathanWalker** [comment]
@nathantsoi this is very helpful! Thank you so much sir!! 👍 
Hoping to finish the rc5 upgrade this weekend and will likely merge this then. 💯

File: tools/tasks/seed/build.js.test.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { join} from 'path';
 
 import { APP_DEST, APP_SRC, TOOLS_DIR, ENABLE_SCSS } from '../../config';
 import { makeTsProject } from '../../utils';
-import { ViewBrokerService } from '../../../src/client/app/frameworks/core/services/view-broker.service';
+import {ViewBroker} from '../../../nativescript/app/app/frameworks/core/utils/view-broker';
```
Comment (NathanWalker) [inline tools/tasks/seed/build.js.test.ts:8]: This should stay from `src/client`... nothing in the build should ever import from nativescript app... at least for now.

File: src/client/app/frameworks/i18n/components/lang-switcher.component.ts [hunk 2]
```diff
@@ -3,11 +3,12 @@ import {Store} from '@ngrx/store';
...
 
 // app
-import {FormComponent, Config, LogService, ILang} from '../../core/index';
+import {Config, LogService, ILang} from '../../core/index';
 import {ElectronEventService} from '../../electron/index';
 import {MultilingualService} from '../index';
+import {Component} from '@angular/core';
 
-@FormComponent({
+@Component({
```
Comment (NathanWalker) [inline src/client/app/frameworks/i18n/components/lang-switcher.component.ts:11]: Not using the decorator here will of course make this component unavailable to the {N} app... just fyi.

Comment (NathanWalker) [inline src/client/app/frameworks/i18n/components/lang-switcher.component.ts:11]: Could probably just use `BaseComponent` here.
I can make these changes once I merge, just kinda talking aloud here ;)

File: tools/tasks/seed/build.js.test.ts (hunk 1 above)
Comment (nathantsoi) [reply tools/tasks/seed/build.js.test.ts:8]: ah yeah, typeahead fail. i should have caught that

File: src/client/app/frameworks/i18n/components/lang-switcher.component.ts (hunk 2 above)
Comment (nathantsoi) [reply src/client/app/frameworks/i18n/components/lang-switcher.component.ts:11]: good call. i didnt know what would happen w/ the BaseComponent as NgModule seemed to replace some of the reason that existed, but was just a guess.

**Comment by nathantsoi** [comment]
awesome, thx for the merge! also, awesome work: https://github.com/NathanWalker/angular2-seed-advanced/commit/43b684e98a87eda3407acd356d9ebb6c531b5c56 :)

**Pull request merged**